modules/apm/src/main/java/org/elasticsearch/telemetry/apm/settings/APMAgentSettings.java

```diff
@@ -31,7 +32,7 @@
  * This class is responsible for APM settings, both for Elasticsearch and the APM Java agent.
  * The methods could all be static, however they are not in order to make unit testing easier.
  */
-class APMAgentSettings {
+public class APMAgentSettings {
```

Comment:
Why the visibility changes for the class and methods?

Reply:
it is the consequence of package structure.
org.elasticsearch.telemetry.apm - the only exported package - which requires that `APMTracer` goes to
org.elasticsearch.telemetry.apm.tracing. I don't want to have  `APMSettings` on the external package. So I moved it into its own package. 
As the `APMSettings` is no no longer in the same package as APMTracer I had to change the scope of methods.
It is safe that it is public, it is on a non exported package